From aaffee9ddce9e6c0c08ada0024a234a0cc458782 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 11 Jul 2012 15:35:33 +0800
Subject: [PATCH 026/281] ENGR00216820 imx6: add libefence.so into PRODUCT
 PACKAGE

libefence is a malloc debugger used for memory
leak detection.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/imx6.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 770c075..c1e576e 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -43,6 +43,7 @@ PRODUCT_PACKAGES += \
 	bash					\
 	taskset					\
 	sqlite3					\
+	libefence				\
 	powerdebug
 
 # Wifi AP mode
-- 
1.8.0

